src/os/FileJournal.cc

```diff
@@ -866,6 +871,45 @@ int FileJournal::check_for_full(uint64_t seq, off64_t pos, off64_t size)
   return -ENOSPC;
 }
 
+int FileJournal::prepare_padding_entry(bufferlist& bl, off64_t& queue_pos)
```

Comment:
If I'm reading this right, if you have some number of entries concatenated together that aren't already aligned, you "wrap" them in a dummy header + some padding to align them?